### DISCUSSION TITLE
quasar.conf.js external ip capacitor

### DISCUSSION THREAD

**Discussion by zanzara** [quasarframework/quasar#9604] [CLI - Capacitor / Cordova modes]
Is there a possibilty to set the external ip in quasar.conf.js i.e  when starting

```
 App · Reading quasar.conf.js
? What external IP should Quasar use? 192.168.0.221
 App · Checking listening address availability (192.168.0.221:8080)...
 App · Extension(@quasar/qenv): Extending quasar.conf...
```

I have to choose it every time, when I start this cmd.

```
quasar dev -m capacitor -T android
```

**Comment by ontwikkelfabriek** [comment]
@zanzara 

You can set the ip/host in the `quasar.conf.js` file
```javascript 
devServer: {
   host:'192.168.0.221'
}
```
See the docs:
https://quasar.dev/quasar-cli/quasar-conf-js#property-devserver

**Reply by zanzara** [reply]
Thx.
I have so much stuff there under `devServer:`  (Cors etc.) that I have might overseen the essential setting ;-)
